diff --git a/src/Makefile.am b/src/Makefile.am
index e66fcb29..9fd58dd8 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -355,7 +355,6 @@ uninstall:
 
 install-exec-hook:
 	$(MKDIR_P) "$(DESTDIR)$(bindir)"
-	$(MKDIR_P) "$(DESTDIR)$(cloudi_logdir)"
 	$(MKDIR_P) "$(DESTDIR)$(cloudi_confdir)"
 	$(MKDIR_P) "$(DESTDIR)$(cloudi_bindir)"
 	$(MKDIR_P) "$(DESTDIR)$(cloudi_prefix)"
@@ -363,8 +362,7 @@ install-exec-hook:
 	$(MKDIR_P) "$(DESTDIR)$(cloudi_prefix)/lib"
 	$(MKDIR_P) "$(DESTDIR)$(cloudi_prefix)/releases"
 	(cd "$(DESTDIR)$(cloudi_prefix)" && \
-     test ! -e etc && $(LN_S) "$(cloudi_confdir)" etc || exit 0 && \
-     test ! -e logs && $(LN_S) "$(cloudi_logdir)" logs || exit 0)
+     test ! -e etc && $(LN_S) "$(cloudi_confdir)" etc || exit 0)
 	$(ESCRIPT) $(top_srcdir)/lib/reltool_util/release
 	test ! -d "$(DESTDIR)$(cloudi_prefix)/erts-$(ERLANG_ERTS_VER)" && \
        mv -f "$(top_builddir)/rel/cloudi/erts-$(ERLANG_ERTS_VER)" \
